app/notify/notify_email/candidate_verify_email_link.0e4b2eaa-ae1f-472a-9293-c2a24f3f8187.md

```diff
@@ -4,4 +4,4 @@ Click the link below to verify your email address
 
 ((verification_link))
 
-This link will expire within 1 hour.
```

Comment:
Same again - this actually does expire after 1 hour, can the notify template be amended and this change removed